refiners/array_axiom_enumerator.cpp

```diff
@@ -142,6 +142,10 @@ ArrayAxiomEnumerator::ArrayAxiomEnumerator(ArrayAbstractor & aa,
 {
   conc_bad_ = solver_->make_term(Not, prop);
   false_ = solver_->make_term(false);
+}
+
+void ArrayAxiomEnumerator::initialize()
```

Comment:
What do you think about adding an `initialized_` flag that is checked in `enumerate_axioms`? It could be an assertion so it's only included in debug mode, but it's just an extra check that the developer didn't forget to initialize.

Reply:
good idea. I am going to do it now.